javascript/grid-ui/package.json

```diff
@@ -61,5 +61,19 @@
 			"last 1 firefox version",
 			"last 1 safari version"
 		]
+	},
+	"devDependencies": {
```

Comment:
All of these need to be at the to be in the top level `package.json` file in the root of the repo